SIAB/spillage/jy_expmt.py

```
@@ -0,0 +1,203 @@
+    def test_coef_init(self):
+        from SIAB.spillage.api import _save_orb
+
+        here = os.path.dirname(__file__)
+        fpath = os.path.join(here, "testfiles/Si/jy-7au/monomer-gamma/OUT.ABACUS")
+        # catch the ValueError
+        with self.assertRaises(ValueError):
+            _coef_init(fpath, [2, 3, 5])
+        out = _coef_init(fpath, [2, 3, 3])
+        # the shape should be, 3 * [2, 3, 3] * nq
+        self.assertEqual(len(out), 3)
+        for l in range(3):
```

Ruff

Ambiguous variable name: `l`

(E741)